build-chat/index.ts

```diff
@@ -0,0 +1,313 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import * as core from '@actions/core'
+import { Octokit } from '@octokit/rest'
+import { WebClient } from '@slack/web-api'
+import * as storage from 'azure-storage'
+import { WritableStreamBuffer } from 'stream-buffers'
+;(async () => {
```

Comment:
Consider placing this all in the onTriggered block of an Action (see ./locker/index.ts), Actions do some important things like log out a particular command to prevent workflow commands from running (otherwise logging untrusted input presents a security vulrability). Also Actions handle logging errors to a github issue, which will ping me if something happens.

Reply:
Using onTriggered now.